.travis.yml

```diff
@@ -1,29 +1,26 @@
-sudo: required 
-dist: trusty 
-language: generic 
-compiler:
-  - gcc
-notifications:
-  email:
-    on_success: always
-    on_failure: always
-#    recipients:
-#      - jane@doe
+sudo: required
+dist: trusty
+language: generic
+
+cache:
+  directories:
+    - $HOME/.ccache
+
 env:
+  global:
+    - CCACHE_DIR=$HOME/.ccache
   matrix:
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - USE_DEB=true ROS_DISTRO="indigo" ROS_REPO=ros NOT_TEST_INSTALL=true
```

Comment:
`USE_DEB` is deprecated; in case of `=true`, you can safely remove it

Reply:
Yeah, thanks. Just noticed it.